.gitignore

```diff
@@ -12,3 +12,5 @@ lerna-*.log
 .DS_Store
 dist
 .cache
+.yalc
```

Comment:
Adding these here to make sure that any yalc files from local development doesn't get accidentally committed in the future